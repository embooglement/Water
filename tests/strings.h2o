print("double quoted");
print('single quoted');
print("double\tquoted\nwith escapes");
print('single\tquoted\nwith escapes');
print("'double quotes'");
print('"single quotes"');
print("\"double quotes with escapes\"");
print('\'single quotes with escapes\'');
