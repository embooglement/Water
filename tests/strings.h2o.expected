double quoted 
single quoted 
double	quoted
with escapes 
single	quoted
with escapes 
'double quotes' 
"single quotes" 
"double quotes with escapes" 
'single quotes with escapes' 

